src/Entity/Phoning/CampaignHistory.php

```diff
@@ -44,6 +59,14 @@
  *         },
  *     },
  * )
+ *
+ * @ApiFilter(SearchFilter::class, properties={
+ *     "campaign.title": "partial",
+ *     "status": "exact",
+ * })
+ * @ApiFilter(PhoningCampaignCallerFilter::class)
```

Comment:
Two filters seem to be similar, what about doing something like that:
```suggestion
 * @ApiFilter(AdherentIdentityFilter::class, properties={
 *     "adherent", 
 *.    "caller",
 * })
```